src/scenes/levelend-scene.ts

```diff
@@ -147,7 +148,7 @@ export class LevelEndScene {
       this.monster.update(deltaTime);
       this.closeButton.draw();
       this.retryButton.draw();
-      if (this.starCount >= 2) {
+      if (!this.isLastLevel && this.starCount >= 2) {
```

Comment:
this is not correct, you only need to check if the game is the last level. The only reason is why you need this.starCount is because in line 105 is incorrect as you are not really getting or validating if it is the last level.
this.current is not an accurate indicator if the game level is last, it is only an indicator to the current level of the user has progressed to. I hope this is clear.

Reply:
there is nothing else to check other then the last level, may it be 0 star 1 star or 3 star there is an option to replay always but there should never be a next level button if it is the last level of the game. we can remove the star count check from the condition.

Reply:
Apologies @woodsjohn-codev 
as discussed with Ashish and we do have a scenario where in any level if the stars are below 2 the next button wont be visible, resolving this comment.
Thanks

Reply:
thank you @ashwinnair-chimple